### PR TITLE
Extending ccr tests

### PR DESCRIPTION
**Pull request by Zartaj0**
1. Added test for claim Arbitrary and Incentivized Chat Request fees. 
2. Updated test structure to work with the forked environment. 
3. Removed unused storage from PushComm Storage contract. 
4. Test for HandleArbitraryReq in core
5. Fixed imports
6. added zero check for the arbitrary req in core

### CODE REVIEW

File: test/CCR/ArbitraryRequest/ArbitraryRequest.t.sol [hunk 1]
```diff
@@ -176,6 +173,17 @@ contract ArbitraryRequesttsol is BaseCCRTest {
             hash // Hash of the VAA being used
         );
 
-        assertEq(pushNttToken.balanceOf(address(coreProxy)), amount);
+        assertEq(pushToken.balanceOf(address(coreProxy)), amount);
+    }
+
+    function test_when_UserTries_ClaimingArbitraryTokens() external {
```
Comment (0xNilesh) [inline test/CCR/ArbitraryRequest/ArbitraryRequest.t.sol:179]: add a assert check for updated protocol_pool_fee as well
in all claiming test functions

Comment (0xNilesh) [inline test/CCR/ArbitraryRequest/ArbitraryRequest.t.sol:179]: add a assert check for updated protocol_pool_fee as well
in all claiming test functions

File: test/CCR/SpecificRequest/CreateChatCCR/CreateChatCCR.tree [hunk 2]
```diff
@@ -17,5 +17,9 @@ CreateChatCCR.t.sol
     │   └── it should Revert
     ├── when deliveryHash is used already
     │   └── it should Revert
-    └── when all checks pass
-        └── it should emit event and update storage
\ No newline at end of file
+    ├── when all checks pass
+    │   └── it should em###it event and update storage
```
Comment (0xNilesh) [inline test/CCR/SpecificRequest/CreateChatCCR/CreateChatCCR.tree:21]: typo

**Review by 0xNilesh — CHANGES_REQUESTED** [review]
1. ArbitraryRequest.tree file should have claiming tokens included as well
2. Specific files for testing handleChatRequestData and claimChatIncentives are missing

File: test/CCR/ArbitraryRequest/ArbitraryRequest.t.sol (hunk 1 above)
Comment (Zartaj0) [reply test/CCR/ArbitraryRequest/ArbitraryRequest.t.sol:179]: But, protocol_pool_fees is not being updated anywhere in claim functions

**Review by 0xNilesh — APPROVED** [review]

**Review by zaryab2000 — APPROVED** [review]

**Pull request merged**